components/trust-manager/development/trust-manager-helm-generator.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: builtin
+kind: HelmChartInflationGenerator
+metadata:
+  name: trust-manager
+name: trust-manager
+repo: https://charts.jetstack.io
+version: 0.9.0
```

Comment:
I see there's version 0.19.0. Can we use the latest?

Reply:
I will try it

Reply:
Done